### PR TITLE
fix: no print unused kcl log

### PR DESCRIPTION
**Pull request by healthjyk**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., design docs, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by healthjyk** [comment]
fix bug of unexpected git cloning log

File: pkg/cmd/build/builders/kcl/kcl_builder.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ import (
 	"kcl-lang.io/kpm/pkg/api"
 	"kcl-lang.io/kpm/pkg/opt"
 
-	"kusionstack.io/kusion/pkg/apis/core/v1"
+	v1 "kusionstack.io/kusion/pkg/apis/core/v1"
```
Comment (SparkYuan) [inline pkg/cmd/build/builders/kcl/kcl_builder.go:19]: Why did you add a new import but didn't use it later?

File: go.mod [hunk 2]
```diff
@@ -69,7 +69,7 @@ require (
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 	kcl-lang.io/kcl-go v0.7.1
 	kcl-lang.io/kcl-plugin v0.5.0
-	kcl-lang.io/kpm v0.6.0
+	kcl-lang.io/kpm v0.6.1-0.20240129094517-4edf455a9653
```
Comment (SparkYuan) [inline go.mod:72]: Is this the official release? We shouldn't import a beta version.

**Review by SparkYuan — APPROVED** [review]
lgtm

**Pull request merged**